t/integration/tasks.py

```diff
@@ -206,7 +206,7 @@ def retry_once(self, *args, expires=60.0, max_retries=1, countdown=0.1):
                      max_retries=max_retries)
 
 
-@shared_task(bind=True, expires=60.0, max_retries=1)
```

Comment:
I don't see expires in documentation of shared_task
Here are some additional confirmation of that
https://stackoverflow.com/questions/32217358/is-there-way-to-specify-expiry-when-creating-a-celery-task
expepires is possible in apply_async